dotCMS/src/main/java/com/dotmarketing/servlets/LoginEditModeServlet.java

```diff
@@ -77,5 +84,14 @@ protected void service(HttpServletRequest request, HttpServletResponse response)
 		}
 		request.getRequestDispatcher("/c").forward(request,response);
 	}
+
+    private String findAssetURI(Contentlet cont) throws Throwable {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)